nixos/modules/installer/cd-dvd/installation-cd-minimal.nix

```diff
@@ -1,14 +1,17 @@
 # This module defines a small NixOS installation CD.  It does not
 # contain any graphical stuff.
 
-{ ... }:
+{ lib, ... }:
 
 {
-  imports =
-    [ ./installation-cd-base.nix
-    ];
+  imports = [
+    ../../profiles/minimal.nix
+    ./installation-cd-base.nix
+  ];
 
-  isoImage.edition = "minimal";
+  documentation.man.enable = lib.mkOverride 500 true;
```

Comment:
```suggestion
  documentation.man.enable = lib.mkDefault true;
```

Reply:
Fixed.

Reply:
Oh... In your variant, there would be a conflict with 
```
documentation.man.enable = mkDefault false;
```
in file `nixos/modules/profiles/minimal.nix`

Returned.